app/src/main/java/bankingapp/LogIn.java

```diff
@@ -13,58 +14,7 @@ public class LogIn {
 
     private String accountNumber;
     private String pinNumber;
```

Comment:
These two fields have no usages, and can be removed. Unless you have plans for them, that is.

Reply:
In fact, since `checkAccount` is self-contained now, consider making Login a static class, and `checkAccount` a static method. That will reduce surprises for someone who wants to use this class without reading it first. They'll avoid wasting time with questions like,

* _Why do I need to instantiate a class in order to call the self-sufficient method `checkAccount`?_
* Or, conversely, _Since I need to instantiate the LogIn class before calling the method, `checkAccount` must have a dependency on something else in the class. What is it?_